clients/src/test/java/org/apache/kafka/common/message/MessageTest.java

```diff
@@ -485,7 +485,7 @@ public void testOffsetCommitRequestVersions() throws Exception {
             if (version == 1) {
                 testEquivalentMessageRoundTrip(version, requestData);
             } else if (version >= 2 && version <= 4) {
-                testAllMessageRoundTripsBetweenVersions(version, (short) 4, requestData, requestData);
+                testAllMessageRoundTripsBetweenVersions(version, (short) 5, requestData, requestData);
```

Comment:
Why are we changing this?

Reply:
`testAllMessageRoundTripsBetweenVersions` exclude the end number so it should pass 5 so as to test version 4